### PR TITLE
Port Aurora Tajaran Sprites and haircuts

### PR DESCRIPTION
**Pull request by Superhats**
Tajaran now have some new sprites courtesy of Aurora

Those should color MUCH better, among other things.

They also get new haircuts — in the genemods tab — featuring **in-built ears**

This also gives them a nice autohiss in the forrrrm of rrolled rrrr's

### CODE REVIEW

File: code/modules/mob/living/autohiss.dm [hunk 1]
```diff
@@ -26,6 +26,12 @@
 					LANGUAGE_UNATHI_YEOSA
 	)
 
+
+/datum/species/tajaran
+	autohiss_basic_map = list(
+			"r" = list("rr", "rrr", "rrrr")
```
Comment (BurgerLUA) [inline code/modules/mob/living/autohiss.dm:32]: Why is this here?

Comment (Afya212) [reply code/modules/mob/living/autohiss.dm:32]: I'm working on a PR with Weezer (Lore Master) for Taj. This may or may not be added but I say wait and see what ends up being decided.

Comment (Superhats) [reply code/modules/mob/living/autohiss.dm:32]: From what I've read of tajaran lore, they've got a mild accent. (Or at least, they do on Aurora). Considering bay has basically no tajaran lore, I don't know if that's still accurate.

Comment (Afya212) [reply code/modules/mob/living/autohiss.dm:32]: Taj lore is being worked on right now, would recommend just removing it now and waiting until then.

**Comment by Superhats** [comment]
According to feedback (and because I've messed my branch up) I'll be reopening the PR without the cosmetic changes, making it 100% sprite changes.
